### ISSUE TITLE
Where can I find  "schema.cql" file ?

### ISSUE THREAD

**Issue by ZhouZiHe** [uber/jaeger#132]
hello , I want  to run the "Jaeger" locally , but I can not fing "schema.cql" .

**Comment by black-adder** [comment]
You can find the cassandra cql file here: https://github.com/uber/jaeger/blob/master/plugin/storage/cassandra/cassandra3v001.cql

You can check out https://github.com/uber/jaeger/blob/master/crossdock/main.go#L125
and https://github.com/uber/jaeger/blob/master/crossdock/main.go#L135

for the flags you'll need to initialize collector and query respectively. We're still working on writing up a guide on how to bring up all the services.

**Comment by yurishkuro** [comment]
I just deleted `.cql` file. The preferred way is to generate it using the shell script https://github.com/uber/jaeger/blob/master/plugin/storage/cassandra/cassandra3v001-schema.sh

**Comment by ZhouZiHe** [comment]
Thanks a lot, ! ! ! 
One more thing I have doubts about  What is the  model "crossdock"  working  for ???

**Comment by black-adder** [comment]
crossdock is a integration testing library that we're using. It brings up collector, query, agent, cassandra, and our language specific clients in docker containers and runs end to end tests on them.

**Comment by ZhouZiHe** [comment]
OK, ,I also met another problem : when I start the module "collector" , an error happened like this 
"Unable to build span handlers","error":"no connections were made when creating the session".
I want to know whether I need to configure Cassandra .

**Comment by yurishkuro** [comment]
`go run ./cmd/collector/main.go -h`

prints all cmd line options supported by the collector. If you're not running Cassandra locally, you can specify where to find it, eg.

```
  -cassandra.keyspace string
    	The Cassandra keyspace for Jaeger data (default "jaeger_v1_local")
  -cassandra.port int
    	The port for cassandra
  -cassandra.servers string
    	The comma-separated list of Cassandra servers (default "127.0.0.1")
```

**Comment by mabn** [comment]
@ZhouZiHe Had the same issue ("no connections were made when creating the session"). The problem was with "-cassandra.connections-per-host" - it defaults to 0. Anything higher works.

**Comment by yurishkuro** [comment]
@mabn connections-per-host should be already fixed by #138

**Comment by badiib** [comment]
Closing issue, seems like it's fixed.
